src/test/java/org/broadinstitute/hellbender/utils/nio/LineIteratorTest.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LineIteratorTest {
```

Comment:
Extend `GATKBaseTest`, rename to have a `*UnitTest` suffix in the class name, and update to reflect the new name for `LineIterator`

Reply:
OK, done.